### PR TITLE
Make ObfuscatedAction collection case insensitive

### PR DESCRIPTION
**Pull request by cristinamanum**
Address one comment missed in previous PR.

### CODE REVIEW

File: src/NuGetGallery/Telemetry/Obfuscator.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ internal static class Obfuscator
         /// </summary>
         internal const string DefaultTelemetryUserName = "ObfuscatedUserName";
 
-        internal static readonly HashSet<string> ObfuscatedActions = new HashSet<string>{
+        internal static readonly HashSet<string> ObfuscatedActions = new HashSet<string>(StringComparer.InvariantCultureIgnoreCase){
```
Comment (joelverhagen) [inline src/NuGetGallery/Telemetry/Obfuscator.cs]: nit: curly on next line

File: tests/NuGetGallery.Facts/Telemetry/ObfuscatorFacts.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace NuGetGallery.Telemetry
+{
+    public class ObfuscatorFacts
+    {
+        [Fact]
+        public void ObfuscatedActionsAreNotCaseSensitive()
```
Comment (joelverhagen) [inline tests/NuGetGallery.Facts/Telemetry/ObfuscatorFacts.cs]: Use turkey "i" to assert the Invariant is used

File: src/NuGetGallery/Telemetry/Obfuscator.cs (hunk 1 above)
Comment (joelverhagen) [reply src/NuGetGallery/Telemetry/Obfuscator.cs]: `OrdinalIgnoreCase` is supposed to be faster

File: tests/NuGetGallery.Facts/Telemetry/ObfuscatorFacts.cs (hunk 2 above)
Comment (joelverhagen) [reply tests/NuGetGallery.Facts/Telemetry/ObfuscatorFacts.cs]: This is a crazy edge case. You can ignore this if you feel like it's not worth testing.

File: src/NuGetGallery/Telemetry/Obfuscator.cs (hunk 1 above)
Comment (cristinamanum) [reply src/NuGetGallery/Telemetry/Obfuscator.cs]: Updated

File: tests/NuGetGallery.Facts/Telemetry/ObfuscatorFacts.cs (hunk 2 above)
Comment (cristinamanum) [reply tests/NuGetGallery.Facts/Telemetry/ObfuscatorFacts.cs]: As we create the routes, I do not think that is worth adding.
